### PR TITLE
Maintenance page for Keycloak migration

### PR DESCRIPTION
**Pull request by grubolsch**
### Added
Added a Maintenance page for the Keycloak migration.
Screenshot 2024-09-23 at 10 51 22

### CODE REVIEW

File: public/503.php [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+            background-color: #009FDE;
+            color: white;
+            padding: 15px;
+            text-align: center;
+        }
+    </style>
+</head>
+<body>
+<div class="container">
+    <img src="/build/assets/Hero-DagNCfZ2.svg" alt="Maintenance Image" class="header-image">
```
Comment (LucWollants) [inline public/503.php:56]: I don't see this asset `Hero-DagNCfZ2.svg`

**Comment by LucWollants** [comment]
For now, this hardcoded approach is fine, but it should move to this folder https://github.com/cultuurnet/publiq-platform/tree/main/resources/views/errors

File: public/503.php (hunk 1 above)
Comment (grubolsch) [reply public/503.php:56]: It already exists, I think it gets deployed there with NPM.
I just copied the path from the homepage.

Comment (LucWollants) [reply public/503.php:56]: Ah it is `Hero.svg`

**Review by LucWollants — APPROVED** [review]
Feel free to use this for now, but afterwards refactor this to a React page like the 404 page.

**Comment by grubolsch** [comment]
> For now, this hardcoded approach is fine, but it should move to this folder https://github.com/cultuurnet/publiq-platform/tree/main/resources/views/errors
I tried that, but the problem is that Inertia and the default Laravel code are not playing nice on this one.
I put files in the views/errors but that didn't do anything, in fact it crashed because it didn't have a resources/views/app.blade.php file.
When I added that, It would show that template, but not the inner templates in errors/. I could see that I tried to load the entire app.blade.php file in some JS which failed in the browser (Inertia?)

All of these things are probably solvable, but I decided it was not worth it for this short time.

**Pull request merged**